sample-lnd.conf

```diff
@@ -519,6 +519,12 @@
 ; intelligence services.
 ; color=#3399FF
 
+; The maximum duration that the server will wait before timing out reading
+; the headers of an HTTP request.
+; Default:
+;   http-header-timeout=5s
+; Example:
```

Comment:
Do we really need an example here? I think this can be more compact by just showing the default value, then we don't need the `Default` label either.

Reply:
I removed the example.